build.gradle.kts

```diff
@@ -257,10 +257,10 @@ mavenPublishing {
 
 dependencies {
     annotationProcessor(libs.nullaway)
+    api(libs.jspecify)
```

Comment:
jspecify appears to be the newer long term replacement for javax annotations, so switched to it. Previously we pulled these annotations in through Guava's jsr305 dependency.